pages/api/group/[gid]/member.ts

```diff
@@ -0,0 +1,82 @@
+import mongoose from 'mongoose';
+import type { NextApiRequest, NextApiResponse } from 'next';
+import {dbConnect} from '@/lib/mongoClient';
+import { responseFormatter } from '@/lib/responseLib';
+import Group from '@/models/Group';
+import  UserGroupRelation  from '@/models/UserGroupRelation';
+
+export default async function groupMemberHandler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+
+  // Connect with MongoDB database
+  try {
+    await dbConnect();
+  } catch (e) {
+    console.log('fail to connect with database', e);
+    return (
+      res
+        .status(500)
+        .json(responseFormatter(false, null, 'Internal Server Error'))
+    );
+  }
+  const {gid} = req.query
+  const group = await Group.findOne({gid: gid});
+
+  switch (req.method) {
+    case 'POST':
```

Comment:
Again, only `admin` or `creator` can make such change, you see the authorization logic comes again.